index.js

```diff
@@ -280,25 +292,44 @@ async function fetchExportFileById(type, envId, id) {
 
 async function fetchAllExportFiles(type, envId, page = 1) {
   const templates = await SF.readExportFiles(type, envId, page);
+
   if (templates.length == 0) {
     if (page == 1) {
-      consola.error(`No export files found in firm ${firmId}`);
+      consola.error("No export files found in firm");
     }
     return;
   }
+
   templates.forEach(async (template) => {
-    fetchExportFileById(type, envId, template.id);
+    const saved = ExportFile.save(type, envId, template.id);
```

Comment:
```suggestion
    const saved = ExportFile.save(type, envId, template);
```
![image](https://github.com/user-attachments/assets/e0eb87ed-9da6-459a-9e3c-698ffdfe513f)

Reply:
Removed the `.id`